.rubocop.yml

```diff
@@ -11,15 +11,9 @@ Layout/AlignHash:
 Layout/EmptyLines:
   Enabled: false
 
-Layout/EmptyLinesAroundClassBody:
```

Comment:
I like having a few lines after the end of a class body, so that I can scroll better in my IDE.

Reply:
I don't get this objection — would be intrigued to understand better. There are other options we can choose — one empty line around the class, etc. Personally I don't like more than 1 as it looks like someone forgot to finish writing the code.

Reply:
Think that applies to Layout/TrailingEmptyLines — totally fine with disabling this rule.